api/ruby/build-your-first-github-app/README.md

```diff
@@ -0,0 +1,19 @@
+This is the sample project built by following the "[Build Your First GitHub App](https://developer.github.com/apps/build-your-first-github-app)" quickstart guide on developer.github.com.
+
+It consists of two different servers: `server.rb` (boilerplate) and `advanced_server.rb` (completed project).
+
+## Install and run
+
+To run the code, make sure you have [Bundler](http://gembundler.com/) installed; then type `bundle install` on the command line.
+
+For the boilerplate project, type `ruby server.rb` on the command line.
+
+For the completed project, enter `ruby advanced_server.rb` on the command line.
+
+Both commands will run the server at `localhost:4567`.
+
+## History
+
+The original author is @DEGoodmanWilson, and the original location is https://github.com/DEGoodmanWilson/app-boilerplate.
```

Comment:
The current state of the repo: https://github.com/DEGoodmanWilson/app-boilerplate contains the initial framework but not all the files presented here in the PR